geth/node/manager.go

```diff
@@ -140,7 +141,11 @@ func (m *NodeManager) StopNode() (<-chan struct{}, error) {
 
 // stopNode stop Status node. Stopped node cannot be resumed.
 func (m *NodeManager) stopNode() (<-chan struct{}, error) {
-	if m.node == nil || m.nodeStarted == nil || m.nodeStopped == nil {
+	if m.node == nil {
```

Comment:
Why do we have direct access to the `NodeManager.node` unguarded by mutex. We might need to create a new issue to refactor this properly, but we can't handle it here.